jwst/flatfield/flat_field.py

```diff
@@ -973,6 +907,9 @@ def spectrograph_flat(wl, s_flat_model,
 
     s_flat_dq : ndarray, 2d, uint32, or None
         The associated data quality array.
+
+    s_flat_err : ndarray, 2d, float32, or None
+        The associated error array.
     """
 
     if s_flat_model is None:
```

Comment:
Should this block return also `s_flat_err = None`?

Reply:
I believe it does?  L941-946?

Reply:
Ah, I see.  A few lines down. That's a bug.  Thanks.  Fixing.